### PR TITLE
[zh]sync pr-wranglers.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/contribute/participate/pr-wranglers.md

### CODE REVIEW

File: content/zh-cn/docs/contribute/participate/pr-wranglers.md [hunk 1]
```diff
@@ -92,6 +92,20 @@ Each day in a week-long shift as PR Wrangler:
...
+<!--
+PR wrangler duties do not apply to localization PRs (non-English PRs). 
+Localization teams have their own processes and teams for reviewing their language PRs. 
+However, it's often helpful to ensure language PRs are labeled correctly, 
+review small non-language dependent PRs (like a link update), 
+or tag reviewers or contributors in long-running PRs (ones opened more than 6 months ago and have not been updated in a month or more).
+-->
+PR 管理者的职责不适用于本地化 PR（非英语 PR）。
+本地化团队有自己的流程和团队来审查其语言 PR。
+但是，其对于确保正确标记语言 PR、审查小型非语言相关 PR（如链接更新），或在长期运行的
```
Comment (tengqm) [inline content/zh-cn/docs/contribute/participate/pr-wranglers.md:105]:
```suggestion
但是，其对于确保被语言 PR 被正确标记，审查与语言无关的小型 PR（如链接更新），或为长期搁置的
```

File: content/zh-cn/docs/contribute/participate/pr-wranglers.md [hunk 2]
```diff
@@ -92,6 +92,20 @@ Each day in a week-long shift as PR Wrangler:
...
+PR（已打开超过 6 个月且一个月或更长时间未更新的）中标记审阅者或贡献者通常很有帮助。
```
Comment (tengqm) [inline content/zh-cn/docs/contribute/participate/pr-wranglers.md:106]:
```suggestion
PR（已打开超过 6 个月且一个月或更长时间未更新的）添加审阅者或贡献者标签通常很有帮助。
```

**Review by ydFu — APPROVED** [review]
/approve

**Pull request merged**